### ISSUE TITLE
Errors while exporting

### ISSUE THREAD

**Issue by Rebivalion** [0xafbf/blender-datasmith-export#16]
Hello, i need help. I'm trying to export my scene to datasmith file but it keeps crashing. 
![error](https://user-images.githubusercontent.com/57173536/67844544-39362e00-fafe-11e9-9012-be81ab7b081c.png)
Thank you for any help !

**Comment by YsengrinWolf** [comment]
Try making your world background just a simple color.

**Comment by 0xafbf** [comment]
Yes, please share your problems in discord as I am more active there

**Comment by ker2x** [comment]
> Try making your world background just a simple color.

it worked for me. thx !

**Comment by joeerito** [comment]
> Try making your world background just a simple color.

not working for me
